### DISCUSSION TITLE
window defender拦截

### DISCUSSION THREAD

**Discussion by anranmengxing** [Meloong-Git/PCL#1949] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/135329120/0815595b-af96-40da-b000-5ec5d2833b3d)
如图所示

**Comment by Pigeon0v0** [comment, marked as answer]
自行设置白名单

**Reply by anranmengxing** [reply]
为啥会误报呢

**Reply by iPodtouch6557** [reply]
我不知道（  科学的解释是快照版没有提交样本信任，然而事实上因为杀软导致的误报都很玄学……甚至比如木马（（（

**Reply by MoYuan-CN** [reply]
百宝箱部分功能导致?

**Reply by iPodtouch6557** [reply]
> 百宝箱部分功能导致?

近期 #666 有人给出了极为详细的原因及解决办法，翻到issue底下去看

**Comment by a10086dd** [comment]
Windows安全中心/病毒和威胁防护/管理设置/添加或删除排除项/添加排除项/文件类型/PCL程序文件名



![IMG_20230610_194622.png](https://github.com/Hex-Dragon/PCL2/assets/94156868/bb0d14f4-a704-4723-8dca-92ef880a0205)

![IMG_20230610_195017.png](https://github.com/Hex-Dragon/PCL2/assets/94156868/7258a1da-7b66-47c9-8ddf-77e68b4fb54f)

![IMG_20230610_194822.png](https://github.com/Hex-Dragon/PCL2/assets/94156868/e9c42ac3-89d6-4cab-b4ae-4aee216931ef)

![IMG_20230610_194527.png](https://github.com/Hex-Dragon/PCL2/assets/94156868/3368598c-43b5-4535-8680-5fcc2e8934e7)

![IMG_20230610_194808.png](https://github.com/Hex-Dragon/PCL2/assets/94156868/4f58312f-912d-4051-ba90-1892878fd302)

**Reply by iPodtouch6557** [reply]
排除项对有些人不管用，比如我的排除项莫名其妙消失了，应该是由于联想自带联想的杀毒软件，导致其被联想垄断了，翻译成人话就是不能再用windows的这一项功能了，只能用联想自带的

**Comment by a10086dd** [comment]
6



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年06月10日 22:19 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [Hex-Dragon/PCL2] window defender拦截 (Discussion #1949) |

排除项对有些人不管用，比如我的排除项莫名其妙消失了，应该是由于联想自带联想的杀毒软件，导致其被联想垄断了，翻译成人话就是不能再用windows的这一项功能了

**Reply by iPodtouch6557** [reply]
请不要用邮件里自带的回复功能来回复垃圾信息，这样不仅阅览起来很麻烦，而且还对解决问题增加一定负担

但是火绒一类的杀毒软件的隔离项是能正常使用的，至于联想一点用都没有就懒得说了吧
